VaticanBAV/et/BAVet129.xml

```diff
@@ -28,171 +28,231 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-            <item xml:id="q16" n="16"><dim unit="leaf">6</dim><locus from="116r" to="123v"></locus></item>
-            <item xml:id="q17" n="17"><dim unit="leaf">8</dim><locus from="124r" to="131v"></locus></item>
-
-        </list>
-      </collation>
-    </supportDesc>
-
-<layoutDesc>
-  <layout columns="2" writtenLines="19"></layout>
-</layoutDesc>
-
-  </objectDesc>
-
-<handDesc>
-<handNote xml:id="h1" script="Ethiopic">
-<date notBefore="1600" notAfter="1699" resp="PRS4805Grebaut PRS9530Tisseran">17th century</date>
-<seg type="ink">Black, red</seg>
-</handNote>
-</handDesc>
-
-<additions>
-<list>
-
-<item xml:id="a1">
-  <desc type="ScribalSignature">The name of the scribe given in various parts of the manuscript is Śǝna Krǝstos.</desc>
-</item>
-
-<item xml:id="a2">
-  <locus target="#131v"/>
-  <desc type="OwnershipNote">The owners of the manuscripts were: Takla Māryām, Za-Giyorgis, Habta Mikāʾel, Yosef and
-  finally ʾErmǝyās who authored the ownership note.</desc>
-  <q xml:lang="gez">ዝመጽሐፍ፡ ዘኤርምያስ፡ ዘአጥረዮ፡ በንዋዩ፡ ዘሠረቆ፡ ወዘፈሐቆ፡ እንበለ፡ ፈቃደ፡ እግዚኡ፡ በሥልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን።</q>
-</item>
-
-</list>
-</additions>
-
-<bindingDesc>
-  <binding>
-    <decoNote xml:id="b1">Two wooden boards covered in leather with a cross impressed on the cover.</decoNote>
-    <decoNote xml:id="b2" type="bindingMaterial">
-       <material key="leather"/>
-       <material key="wood"/>
-    </decoNote>
-  </binding>
-</bindingDesc>
-
-</physDesc>
-
-<additional>
-                      <adminInfo>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">131</measure>
+                                    <measure unit="quire">17</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>200</height>
+                                        <width>176</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <list>
+                                        <item xml:id="q1" n="1">
+                                            <dim unit="leaf">7</dim>
+                                                <!-- Data don't match.
+                                            The quire structure illustrated in the catalogues gives a total number of 133 ff., while the catalogues 
+                                        state that the manuscript has 131 ff. In order to reconcile the two data, one can assume that the first 
+                                        two folia are blank and were not counted by the cataloguers (MKr-MV) -->
```

Comment:
Thank you for the suggestion about `<note/>`. Regarding the loci, in principle I try to give them every time the quire sequence is clear or can be easily reconstructed, but in this very case I preferred not to give loci because data are inconsistent and we do not know exactly the origin of the divergence.